tiled_layers/tile-cache/README.md

```diff
@@ -1,30 +1,39 @@
-# Tile cache
+# ArcGIS tiled layer (tile cache)
```

Comment:
We should hold off on this readme title change since it would require updating the sample code etc: Mike has confirmed this following a similar question from the iOS. 
```suggestion
# Tile cache
```